lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -175,9 +175,28 @@ function distributeTwoNodes(firstNode, secondNode) {
   // the new operands under addition, now products of terms
   const newArgs = [];
 
+  // if exactly one group contains at least one fraction, multiply the
+  // non-fraction group into the numerators of the fraction group
+  if ([firstArgs, secondArgs].filter(hasFraction).length === 1) {
+    const firstArgsFraction = hasFraction(firstArgs);
```

Comment:
slight preference for naming this `firstArgsHasFraction` (to make it clearer it's a boolean) but I can see that it's kinda long, so it's cool if you'd prefer not